pkg/report/report.go

```diff
@@ -83,7 +83,7 @@ func NewReport(precision string) Report { return newReport(precision) }
 
 func newReport(precision string) *report {
 	r := &report{
-		results:   make(chan Result, 16),
+		results:   make(chan Result, 65536),
```

Comment:
How this impacts the results? If it doesn't then please remove.

Reply:
It does not. Will remove.